### ISSUE TITLE
incorrect generation of texts entities

### ISSUE THREAD

**Issue by xpr0gamers** [mrbandler/cds2types#33]
For all entities with attributes, who are mapped with the annotation "localized", the sap cap framework generations a specific database table. 
For example (test folder):
```
entity Books : managed {
    key ID       : Integer;
        title    : localized String(111);
        descr    : localized String(1111);
        longdesc : localized String(1111111);
        author   : Association to Authors;
        genre    : Association to Genres;
        stock    : Integer;
        price    : Decimal(9, 2);
        currency : Currency;
        ![/part1/part2]: String(23) default 'test';
}

entity Genres : sap.common.CodeList {
    key ID       : Integer;
        parent   : Association to Genres;
        children : Composition of many Genres
                       on children.parent = $self;
}
```
The framework (sap cap) create a database table "(namespace)_Books_texts". 
"Cds2types" generate a interface named "ITexts", this is incorrect. 
"ITexts" is not a unique name specifcly, when more than one texts tables exists. 
![image](https://user-images.githubusercontent.com/25172624/176997137-83f86110-a072-4cf3-b3d7-2afdf7316d5a.png)

In additon to that, some texts tables are missing in enum SanitizedEntity and enum Entity. 
```
    export enum SanitizedEntity {
        Address = "Address",
        ArrayUsingEntity = "ArrayUsingEntity",
        CompositoinField = "CompositoinField",
        Authors = "Authors",
        Books = "Books", --> Books_texts
        Texts = "Texts",
        EntityWithSlashes = "EntityWithSlashes",
        Genres = "Genres", --> Genres_texts
        Name = "Name",
    }
```
My proposel is, texts tables should be called "EntityName_texts".
```
    export enum SanitizedEntity {
        EntityWithSlashes = "EntityWithSlashes",
        ArrayUsingEntity = "ArrayUsingEntity",
        Books = "Books",
        Name = "Name",
        Address = "Address",
        Authors = "Authors",
        Genres = "Genres",
        CompositoinField = "CompositoinField",
        Books_texts = "Books_texts",
        Genres_texts = "Genres_texts"
    }
```

**Comment by HeneryHawk** [comment]
Hi @xpr0gamers 

this will be resolved with the pr #27.

Regards
Simon

**Comment by HeneryHawk** [comment]
Hi @xpr0gamers,

the pr #27 was merged today and will be included in the next major release.

Regards
